src/reachy2_sdk/orbita/orbita2d.py

```diff
@@ -134,6 +134,41 @@ def send_goal_positions(self) -> None:
         self._outgoing_goal_positions = {}
         self._stub.SendCommand(command)
 
+    def set_speed_limits(self, speed_limit: float | int) -> None:
+        """Set a speed_limit as a percentage of the max speed on all motors of the actuator"""
+        super().set_speed_limits(speed_limit)
+        speed_limit = speed_limit / 100.0
```

Comment:
This is also used in orbita3d. move to super() ?

Reply:
Didn't manage to properly modify this variable in the super()